app/ui/client/views/app/room.js

```diff
@@ -224,6 +224,34 @@ function roomMaxAge(room) {
 	}
 }
 
+async function createFileFromUrl(url) {
+	let response;
+	try {
+		response = await fetch(url);
+	} catch (error) {
+		if (error.toString().includes('TypeError')) {
```

Comment:
When is this triggered? Can we grant `url` type in the outer scope?

Reply:
@tassoevan https://github.com/RocketChat/Rocket.Chat/blob/fix-clipboard-drag/app/ui/client/views/app/room.js#L599 I Check this here